modules/ROOT/attachments/specs/api.yaml

```diff
@@ -0,0 +1,734 @@
+        language:
+          type: string
+          description: |-
+            Language used to render the notice
+             * Notice labels: use the specified 
+             * Notice content: use the specified language when it exists in the XML notice otherwise use the first notice original language
+          enum:
+            - bg
+            - cs
+            - da
+            - de
+            - el
+            - en
+            - es
+            - et
+            - fi
+            - fr
+            - ga
+            - hu
+            - it
+            - lt
+            - lv
+            - mt
+            - nl
+            - pl
+            - pt
+            - ro
+            - sk
+            - sl
+            - sv
+            - hr
+        format:
+          type: string
+          description: Output format
+          enum:
+            - PDF
+            - HTML
+        textToHighlight:
+          type: string
+          description: Highlight specific text in the notice rendering
+        summary:
+          type: boolean
+          description: Render notice by displaying only the specific elements
+      description: Notice Render Request
+    EsentoolNoticeDto:
+      type: object
+      properties:
+        id:
+          type: string
+          description: The notice identifier (without the version).
+          format: uuid
+          example: 3fa85f64-5717-4562-b3fc-2c963f66afa6
+        versionId:
+          type: string
+          description: The version identifier.
+          example: '01'
+        procedureId:
+          type: string
+          description: The procedure identifier.
+          format: uuid
+          example: 3fa85f64-5717-4562-b3fc-2c963f66afa5
+        status:
+          $ref: '#/components/schemas/NoticeStatus'
+        updatedAt:
+          type: string
+          description: The notice updated date/time.
+          format: date-time
+          example: '2022-01-01T13:15:55.89Z'
+        submittedAt:
+          type: string
+          description: The notice submitted date/time.
+          format: date-time
+          example: '2022-01-01T13:15:55.89Z'
+        publicationId:
+          type: string
+          description: The notice publication identifier.
+        publicationDate:
+          type: string
+          description: The notice publication date.
+          format: date
+        noticeType:
+          type: string
+          description: The notice notice type.
+        legalBasis:
+          type: string
+          description: The notice legal basis code.
+        languages:
+          type: array
+          description: The notice languages.
+          items:
+            type: string
+            description: The notice languages.
+        noticeAuthor:
+          type: string
+          description: The notice author.
+          example: example@email.eu
+        expectedPublicationDate:
+          type: string
+          description: The notice expected publication date
```

Comment:
The notice expected publication date as assigned by the Publications Office based on the OJ S calendar

Reply:
Done